lib/pairing_matrix/commit_reader.rb

```diff
@@ -23,10 +23,16 @@ def authors(since)
       end.compact.reject(&:empty?)
     end
 
+    def titleize(name)
+      name.gsub(/\w+/) do |word|
+        word.capitalize
```

Comment:
What happens if the name has space? e.g [Ajit Singh/Suman]

Reply:
Hi @ajitsing,
It will be fine. Please check the attached screenshot
<img width="317" alt="screenshot 2018-03-21 00 14 44" src="https://user-images.githubusercontent.com/15261043/37676092-fb01957e-2c9c-11e8-9099-c80686dcfeaa.png">